sdk/core/Azure.Core/samples/Diagnostics.md

```diff
@@ -39,9 +39,12 @@ using AzureEventSourceListener traceListener = AzureEventSourceListener.CreateTr
 
 ### Enabling content logging
 
-By default only URI and headers are logged. To enable content logging, set the `Diagnostics.IsLoggingContentEnabled` client option:
+By default only URI and headers are logged. To enable content logging, set the `Diagnostics.IsLoggingContentEnabled` client option and specify a minimum log level of `Verbose`.
 
 ```C# Snippet:LoggingContent
+// Content is logged as Verbose event level
```

Comment:
So we can't directly change the md file as this gets generated after running:
```
eng\scripts\Update-Snippets.ps1
```

Instead, you'll need to update the c# [file](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/tests/samples/LoggingSamples.cs#L58) that contains this sample, and then regenerate the md file.